sdk/src/Services/S3/Custom/_bcl+coreclr+pcl/Transfer/Internal/DownloadCommand.cs

```diff
@@ -55,6 +55,8 @@ internal partial class DownloadCommand : BaseCommand
             (WebExceptionStatus)12,
             (WebExceptionStatus)1,
             (WebExceptionStatus)3,
+            (WebExceptionStatus)14, //timeout
```

Comment:
I am not sure if this is the correct behavior.  If we retry on timeout with delay, in the worst case scenario, we would be waiting for a very long time before we give up.  Is that the behavior we want?